docs/administration/datagrid/fields.md

```diff
@@ -0,0 +1,83 @@
+# Fields
+
+Fields are the building blocks of a datagrid. They are used to define the columns of the datagrid and to specify how the data should be displayed.
+
+## Configuration
+
+Fields are defined by the `Datagrid` class using the `add(string $name, array $options)`, `update(string $name, array $options)` and `remove(string $name)` methods. The `$name` parameter is the name of the field and the `$options` parameter is an array of options that define the behavior of the field.
+
+In case you are using `OrmAdapter`, you can use dot notation to access nested properties of the entity. Here are some examples of how to define field names:
+
+- `$datagrid->add('domainId')` - This will fetch the `domainId` property of the entity.
```

Comment:
thinking... wouldn't be useful to provide a "syntax sugar" to quickly render text as the edit link? 
Now I have to write the custom template. 

something like

```
 $datagrid->add('name', [
            'with_detail_link' => true,
```

Reply:
We can add this functionality later